src/vs/workbench/contrib/terminalContrib/suggest/browser/terminalSuggestAddon.ts

```diff
@@ -304,6 +304,7 @@ export class SuggestAddon extends Disposable implements ITerminalAddon, ISuggest
 		// editor and how it interacts with inline completions. This object is tracked and reused as
 		// it may change on input.
 		this._refreshInlineCompletion();
+		this._addPropertiesToInlineCompletionItem(completions);
```

Comment:
Can this move into the `_refreshInlineCompletion` function?

Reply:
no because of https://github.com/microsoft/vscode/blob/2f6f7ef299a50340c4e58d506ba887131d45d506/src/vs/workbench/contrib/terminalContrib/suggest/browser/terminalSuggestAddon.ts#L542-L552

Reply:
although I guess we could specify whether we want to show completions in a parameter that we pass in?